### PR TITLE
"Misplaced words or phrases" - Grammarly

### PR DESCRIPTION
**Pull request by Tyler490**

### CODE REVIEW

**Comment by NSAntoine** [comment]
are you serious bro

**Comment by Cryptiiiic** [comment]
Please attach actual code contributions to this pull

File: README.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ Latest **stable** compiled version can be found [here](https://github.com/m1stad
 
 For A12-A14, and WiFi-only iPad restores - use the latest beta build for your platform [here](https://nightly.link/m1stadev/futurerestore/workflows/ci/test).
 
-**There are currently no pre-compiled beta builds for Windows.**
+**There currently are no pre-compiled beta builds for Windows.**
```
Comment (CoocooFroggy) [inline README.md:10]:
```suggestion
**Currently, there are no pre-compiled beta builds for Windows.**
```

**Comment by CoocooFroggy** [comment]
Also no

**Comment by Tanbeer191** [comment]
They all make sense, this is unnecessary as hell.

**Comment by ScattyNatalie** [comment]
How is this even still open

**Comment by NSAntoine** [comment]
how about we delete the readme 💯

**Comment by Cryptiiiic** [comment]
Because I’m at work @Gymxo

**Pull request closed without merging**